filesystem/btrfs_stream.ksy

```diff
@@ -0,0 +1,151 @@
+meta:
+  id: btrfs_stream
+  application: Btrfs
+  file-extension: btrfs-stream
+  endian: le
+  license: CC0-1.0
+doc: |
+  This spec can be used to disassemble raw Btrfs Send stream format
+  version 1 from the output of `btrfs send` command.
+  See also the `btrfs receive --dump` command.
+doc-ref: https://btrfs.wiki.kernel.org/index.php/Design_notes_on_Send/Receive
+
+seq:
+  - id: header
+    type: send_stream_header
+  - id: commands
+    type: send_command
+    repeat: eos
+types:
+  send_stream_header:
+    seq:
+      - id: magic
+        contents: ['btrfs-stream', 0x00]
+      - id: version
+        type: u4le
+  send_command:
+    seq:
+      - id: size
+        type: u4le
+      - id: type
+        type: u2le
+        enum: command
+      - id: checksum
+        size: 4
```

Comment:
We don't currently have automatic mechanism for checksum checks/calculation (see https://github.com/kaitai-io/kaitai_struct/issues/81), so I would suggest adding description as a documentation entry:

```suggestion
      - id: checksum
        size: 4
        doc: CRC32 checksum of a whole send command, including the header, with this attribute set to 0.
```

Reply:
Description added